interface-definitions/include/babel/interface.xml.i

```diff
@@ -0,0 +1,187 @@
+          <list>enable disable</list>
+        </completionHelp>
+        <valueHelp>
+          <format>default</format>
+          <description>Enable on wired interfaces, and disable on wireless interfaces</description>
+        </valueHelp>
+        <valueHelp>
+          <format>enable</format>
+          <description>Enable split horizon processing</description>
+        </valueHelp>
+        <valueHelp>
+          <format>disable</format>
+          <description>Disable split horizon processing</description>
+        </valueHelp>
+        <constraint>
+          <regex>(default|enable|disable)</regex>
+        </constraint>
+      </properties>
+      <defaultValue>default</defaultValue>
+    </leafNode>
+    <leafNode name="hello-interval">
+      <properties>
+        <help>Time between scheduled hellos</help>
+        <valueHelp>
+          <format>u32:20-655340</format>
+          <description>Milliseconds</description>
+        </valueHelp>
+        <constraint>
+          <validator name="numeric" argument="--range 20-655340"/>
+        </constraint>
+      </properties>
+      <defaultValue>4000</defaultValue>
+    </leafNode>
+    <leafNode name="update-interval">
+      <properties>
+        <help>Time between scheduled updates</help>
+        <valueHelp>
+          <format>u32:20-655340</format>
+          <description>Milliseconds</description>
+        </valueHelp>
+        <constraint>
+          <validator name="numeric" argument="--range 20-655340"/>
+        </constraint>
+      </properties>
+      <defaultValue>20000</defaultValue>
+    </leafNode>
+    <leafNode name="rxcost">
+      <properties>
+        <help>Base receive cost for this interface</help>
+        <valueHelp>
+          <format>u32:1-65534</format>
+          <description>Base receive cost</description>
+        </valueHelp>
+        <constraint>
+          <validator name="numeric" argument="--range 1-65534"/>
+        </constraint>
+      </properties>
+    </leafNode>
+    <leafNode name="rtt-decay">
+      <properties>
+        <help>Decay factor for exponential moving average of RTT samples</help>
+        <valueHelp>
+          <format>u32:1-256</format>
+          <description>Decay factor, in units of 1/256</description>
+        </valueHelp>
+        <constraint>
+          <validator name="numeric" argument="--range 1-256"/>
+        </constraint>
+      </properties>
+      <defaultValue>42</defaultValue>
+    </leafNode>
+    <leafNode name="rtt-min">
+      <properties>
+        <help>Minimum RTT starting</help>
+        <valueHelp>
+          <format>u32:1-65535</format>
+          <description>Milliseconds</description>
+        </valueHelp>
+        <constraint>
+          <validator name="numeric" argument="--range 1-65535"/>
+        </constraint>
+      </properties>
+      <defaultValue>10</defaultValue>
+    </leafNode>
+    <leafNode name="rtt-max">
+      <properties>
+        <help>Maximum RTT starting</help>
```

Comment:
I'm confused by this wording. What does "starting" refer to?

Reply:
It is a leftover of copypasta. Now fixed.